presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -65,10 +65,26 @@ public class NativeExecutionSystemConfig
     private static final String SYSTEM_MEMORY_GB = "system-memory-gb";
     private static final String QUERY_MEMORY_GB = "query.max-memory-per-node";
     private static final String USE_MMAP_ALLOCATOR = "use-mmap-allocator";
+    // Memory arbitration related configurations.
+    // Set the memory arbitrator kind. If it is empty, then there is no memory
+    // arbitration, when a query runs out of its capacity, the query will fail.
+    // If it set to "SHARED" (default), the shared memory arbitrator will try
+    // to trigger disk spilling to reclaim memory so the query can run through
+    // completion.
```

Comment:
// If it set to "SHARED" (default), the shared memory arbitrator will be used to conduct arbitration and try
    // to trigger disk spilling to reclaim memory so the query can run through
    // completion.